pages/Batches/Overview.md

```diff
@@ -59,3 +59,10 @@ consists of a URL and an object of DSNP message column name/value pairs.  Exampl
 ```
 
 Not all fields of a DSNP data type are added to the Bloom filters.  Generally speaking, content hashes and URIs are omitted, whereas any field containing a socialAddress or other meaningful text is included in the Bloom filter.  For more detail, see [See the Messages Overview](/Messages/Overview).
+
+## Validation
+### Batch File Validation
+Batch files are hashed using [keccak-256](https://en.wikipedia.org/wiki/SHA-3) so that it can later be used to verify the integrity of the file. The [keccak-256](https://en.wikipedia.org/wiki/SHA-3) hash of the file is submitted as part of a batch [announcement](/Messages/Announce) that result in a [DSNPBatch](/Messages/Announce#announcing-dsnp-events) Ethereum log event.
+
+### Batch File DSNP Type Validation
+Each batch file consist of one type of [DSNP announcement](/Messages/Overview#dsnp-announcement-formats). Therefore, the file columns should correspond to the format listed in the DSNP announcement dsnpData field. For example, if the file claims to be of DSNP type Broadcast, then the file is expected to include the following columns: fromId, contentHash, uri.  As long as the batch file hash can be verified, order of dsnpData fields are irrelevant.
```

Comment:
This looks really good, just one typo and suggestion to put the field names in code format.
```suggestion
Each batch file consists of one type of [DSNP announcement](/Messages/Overview#dsnp-announcement-formats). Therefore, the file columns should correspond to the format listed in the DSNP announcement dsnpData field. For example, if the file claims to be of DSNP type Broadcast, then the file is expected to include the following columns: `fromId, `contentHash`, `uri`.  As long as the batch file hash can be verified, order of dsnpData fields are irrelevant.
```

Reply:
You might also add that you won't be able to read the file if the type doesn't match the schema in the file, or something like that, so there isn't any other validation necessary.